Describe limitations of the process instance migration for joining parallel gateway (#5807)
camunda/camunda-docs: versioned_docs/version-8.7/components/concepts/process-instance-migration.md

- Multi-instance body limitations:
  - Each child instance of a multi-instance body should be migrated separately because they belong to another process instance.
  - It is not possible to migrate a parallel multi-instance body to a sequential multi-instance body and vice versa.
- Joining parallel gateway limitations:
  - All taken sequence flow from the source needs to be mapped in the target definiton

>   - All taken sequence flow needs to be mapped to the same type of gateway

Reviewer: Singular / plural is mixed up here.

Do you want to say:
* All taken sequence flow**s** have to be mapped to the target definition?

To be honest: these two points are hard to understand. I fail to understand what you are trying to say :-)

>   - The parallel gateway needs to have the same number of incoming sequence flows in before and after migration

Reviewer: ```suggestion
  - Parallel gateways must have the same number of incoming sequence flows before and after migration
```

  - All taken sequence flows that were connected to the parallel gateway in the target definition have to be still directly connected to the parallel gateway
  - The way to migrate taken flows connected to the joining parallel gateway:

>   ![image](https://github.com/user-attachments/assets/2159e991-895e-40f8-8aeb-308fa42d48e6)

Reviewer: Image not included in PR
